package.json

```diff
@@ -28,11 +28,12 @@
     "iot",
     "device",
     "IoT Edge",
-    "simulator"
+    "device provisioning service"
```

Comment:
Only 5 keywords are allowed, so remove the simulator one.

Reply:
dps better or the full string?

Reply:
updated the description and keywords